### ISSUE TITLE
Set up transifex

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/twofactor_webauthn#78]
**Is your feature request related to a problem? Please describe.**

This app needs translation.

**Describe the solution you'd like**

Set up transifex. Copy existing strings from the twofactor_u2f resource as there is a large overlap. 

**Describe alternatives you've considered**

n/a

**Additional context**

https://github.com/nextcloud/docker-ci/pull/351

**Comment by nickvergessen** [comment]
@rakekniven @Valdnet  is it somehow possible to rename the resource in transifex?

**Comment by Valdnet** [comment]
I do not know. I do not have access to edit these sources in Transifex.

**Comment by rakekniven** [comment]
Hello, it is not really clear to me what should be achieved?

Currently we have the following twofactor resources at Transifex:
<img width="995" alt="Screenshot 2022-02-09 at 12 49 22" src="https://user-images.githubusercontent.com/2069590/153195670-d4d56751-3fe0-461a-9017-fc25e1168c34.png">

The following actions can be done to a resource:
<img width="682" alt="Screenshot 2022-02-09 at 12 48 53" src="https://user-images.githubusercontent.com/2069590/153195896-221aab1f-5272-46a3-a538-992c410c9cdc.png">

So renaming of a resource is possible. NOT copying.
Please keep the daily sync at 5AM CET in mind when renaming is done.

After reading the initial post once more I think: Why not adding this resource like any other one? 
Change for docker-ci has already been done. The resource is not big (29 Strings) and translators will quickly do the work.

**Comment by nickvergessen** [comment]
> Why not adding this resource like any other one?
Change for docker-ci has already been done. The resource is not big (29 Strings) and translators will quickly do the work.

Fine by me. Just wanted to save you some time, but also with an upcoming follow up release of u2f, maybe that is the easier thing to do

**Comment by nickvergessen** [comment]
> Please keep the daily sync at 5AM CET in mind when renaming is done.

https://github.com/nextcloud/docker-ci/pull/351 that is where we come from :D

**Comment by rakekniven** [comment]
> that is where we come from :D

Read this seconds ago :-)

**Comment by ChristophWurst** [comment]
The twofactor_webauthn is based on twofactor_u2f, therefore lots of identical strings are used in both apps and I thought it makes sense to copy them to avoid some double translation for twofactor_webauthn that was already done for _u2f.

BUT _u2f won't be shut down yet. So please *do not* rename.

**Comment by rakekniven** [comment]
@ChristophWurst Understood your thoughts, but copying is not implemented by TX unfortunately.

> So please _do not_ rename.
Fully agree 😄

**Comment by ChristophWurst** [comment]
Got it. Then we have to start from scratch. Luckily there aren't too many strings to translate :)

**Comment by Valdnet** [comment]
Translators can always take advantage of the translation suggestions for individual strings. One of the works is just to click through text strings 😁.

**Comment by rakekniven** [comment]
One thing I wonder is that https://github.com/nextcloud/twofactor_webauthn/tree/main/l10n already contains translations?
A look at https://github.com/nextcloud/twofactor_webauthn/blob/main/l10n/de_DE.js shows the source is `twofactor_u2f`.
Did you copy these files?

**Comment by ChristophWurst** [comment]
Yes, the main branch was taken from U2F. It's in fact identical with the latest twofactor_u2f master branch until https://github.com/nextcloud/twofactor_webauthn/pull/74 is done.

**Comment by ChristophWurst** [comment]
Transifex can now be enabled for this repo

**Comment by rakekniven** [comment]
> Yes, the main branch was taken from U2F. It's in fact identical with the latest twofactor_u2f master branch until #74 is done.

Ok, so I recommend to empty `l10n` directory. Not every language is in good shape. By deleting existing files you prevent app from shipping wrong translations. We had this some weeks ago. Maybe @nickvergessen will remember 😉 .

Have a nice weekend.
